### PR TITLE
refactor RCCL

### PR DESCRIPTION
**Pull request by SwRaw**

### CODE REVIEW

File: docs/index.rst
Comment (randyh62) [inline docs/index.rst:1]: Needs a gutter for space between tiles. 
.. grid:: 2
  :gutter: 3

**Review by randyh62 — APPROVED** [review]
Left a comment. Otherwise looks good to me.

**Comment by BertanDogancay** [comment]
Please use 'squash and merge' option when merging the PR.

File: docs/index.rst
Comment (SwRaw) [reply docs/index.rst:1]: gutter is needed to control space between multiple tiles. It doesn't have any effect on a single tile.

**Pull request merged**